packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/list-projects-area/useProjectsTableColumns.tsx

```diff
@@ -14,15 +14,17 @@ import { useData } from '@eclipse-sirius/sirius-components-core';
 import Link from '@mui/material/Link';
 import { MRT_ColumnDef } from 'material-react-table';
 import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
 import { Link as RouterLink } from 'react-router-dom';
 import { GQLProject } from './useProjects.types';
 import { UseProjectsTableColumnsValue } from './useProjectsTableColumns.types';
 import { projectsTableColumnCustomizersExtensionPoint } from './useProjectsTableColumnsExtensionPoints';
 
 export const useProjectsTableColumns = (): UseProjectsTableColumnsValue => {
+  const { t } = useTranslation('sirius-web-application', { keyPrefix: 'useProjectsTableColumns' });
   let columnDefinitions: MRT_ColumnDef<GQLProject>[] = [
     {
-      header: 'Name',
+      header: t('name'),
```

Comment:
This key seems not taken into account (visible on `/projects`):
<img width="1621" height="309" alt="Image" src="https://github.com/user-attachments/assets/d244c709-2a21-43a0-b222-4c7fb6076120" />

Creating the i18next context in debug mode gives me this kind of warning in my console:
`key "useTableTranslation.actions" for languages "fr, en" won't get resolved as namespace "sirius-components-tables" was not yet loaded This means something IS WRONG in your setup. You access the t function before i18next.init / i18next.loadNamespace / i18next.changeLanguage was done`.

Maybe I missed something when I have added the i18n support.

NOTE: The debug mode can be activated passing `debug: true` to `i18nextOptions` in `useCreateI18nInstance.ts`